src/release-pr.ts

```diff
@@ -329,7 +329,11 @@ export class ReleasePR {
 
   static addPathStatic(file: string, path?: string) {
     if (path === undefined) {
-      return file;
+      if (file.startsWith('/')) {
```

Comment:
Does it make sense to also check for `\`?

Reply:
Sure! I was originally getting these back from the github API, which always returns `/`, but I could see someone trying to do a path normalization on windows.